org.coolreader_ng.crqt-ng.yml

```diff
@@ -0,0 +1,93 @@
+app-id: org.coolreader_ng.crqt-ng
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: crqt
+rename-desktop-file: crqt.desktop
+rename-appdata-file: crqt.appdata.xml
+rename-icon: crqt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host:ro
```

Comment:
this is a very broad filesystem permission

Reply:
This is a program for reading e-books, it is understood that they can be anywhere on the computer's file system.
Should I restrict access to the file system in the form --filesystem=xdg-documents:ro ?

Reply:
Since the app use Qt it should use the native file dialog (I think it does) in that case it doesn't need permissions.

Reply:
Ok, thanks.